tests/util/providers.ts

```diff
@@ -69,14 +69,11 @@ export const provideWeb3Api = async (port: number, protocol: "ws" | "http" = "ht
 };
 
 export const providePolkadotApi = async (port: number) => {
-  const provider = new WsProvider(`ws://localhost:${port}`);
-  return {
-    provider,
-    apiPromise: await ApiPromise.create({
-      provider: provider,
-      typesBundle: typesBundle as any,
-    }),
-  };
+  return await ApiPromise.create({
+    initWasm: false,
```

Comment:
what's this flag about?

Reply:
Disable wasm wrapper, not needed in our case:
https://github.com/polkadot-js/api/blob/82e15dc36de7e3b6e95f05b9a5447d7e1e4ecba3/packages/api/src/types/index.ts#L39